doc/en/user/source/tutorials/GetFeatureInfo/geojson.rst

```diff
@@ -0,0 +1,40 @@
+GeoJSON output format
+======================
+
+The default GeoJSON output doesn't rely on free marker templates to be produced, nonetheless it is possible to customize the output using them.
+GeoServer will lookup for json templates following the same rules defined for the hmtl output. The only difference regards the file names of the templates which have to be named appending ``_json`` to the name, as below:
```

Comment:
Not the only difference, you also needs to have all of them.

Reply:
Also something must be said about multi-layer GetFeatureInfo requests in combination with templated output.

Reply:
fixed